### PR TITLE
Disable System.Diagnostics.Tests.EventLogSourceCreationTests

### PR DESCRIPTION
**Pull request by safern**
Related to: https://github.com/dotnet/runtime/issues/36135

cc: @dotnet/runtime-infrastructure @Anipik

### CODE REVIEW

File: src/libraries/System.Diagnostics.EventLog/tests/EventLogTests/EventLogSourceCreationTests.cs [hunk 1]
```diff
@@ -11,6 +11,7 @@ public class EventLogSourceCreationTests
     {
         [Trait(XunitConstants.Category, "EventLog")] // Unreliable Win32 API call
```
Comment (safern) [inline src/libraries/System.Diagnostics.EventLog/tests/EventLogTests/EventLogSourceCreationTests.cs:12]: @Anipik was the intention for this trait to be ignored on CI and regular runs? If so we should change it to "IgnoreForCI"

Comment (ViktorHofer) [reply src/libraries/System.Diagnostics.EventLog/tests/EventLogTests/EventLogSourceCreationTests.cs:12]: That was my fault: https://github.com/dotnet/runtime/commit/48a222cbbbee6c39d622a9daee86f8f185263d5f 😢.

Comment (safern) [reply src/libraries/System.Diagnostics.EventLog/tests/EventLogTests/EventLogSourceCreationTests.cs:12]: oh, haha. I'll update the PR 😄

Comment (safern) [reply src/libraries/System.Diagnostics.EventLog/tests/EventLogTests/EventLogSourceCreationTests.cs]: Fixed.

**Comment by safern** [comment]
Spoke with @Anipik offline. We will disable with `ActiveIssue` and keep the issue open as it seems like the failures are related to machine state, so he'll investigate and look to how fix the tests.

**Review by Anipik — APPROVED** [review]

**Pull request merged**